### DISCUSSION TITLE
Record a player and then have a BOT play it back with SourceMod plugins?

### DISCUSSION THREAD

**Discussion by manicogaming** [saul/demofile#306] [Questions]
I was wondering if it is at all possible to create a record of a players position, current weapons, current angles, current buttons using demofile and then play it back on a BOT using SourceMod plugins.

Here's the plugin I'm thinking of.
[https://forums.alliedmods.net/showthread.php?t=164148](url)

**Comment by manicogaming** [comment]
I figured out how to get the position, weapons and angles of a player on each tick, but I don't know how to the current buttons yet.

I'm parsing GOTV demos only.

**Reply by Dastaan2k** [reply]
Could you please share how did you managed to get player's position on each tick ?
